joint_names: l_shoulder_pan_joint l_shoulder_lift_joint l_upper_arm_roll_joint l_elbow_flex_joint l_forearm_roll_joint l_wrist_flex_joint l_wrist_roll_joint

l_shoulder_pan_joint:
  joint: l_shoulder_pan_joint
  goal_reached_threshold: 0.01
  pid:
    p: 2400.0
    d: 18.0
    i: 800.0
    i_clamp: 4.0

l_shoulder_lift_joint:
  joint: l_shoulder_lift_joint
  goal_reached_threshold: 0.01
  pid:
    p: 1200.0
    d: 10.0
    i: 700.0
    i_clamp: 4.0

l_upper_arm_roll_joint:
  joint: l_upper_arm_roll_joint
  goal_reached_threshold: 0.01
  pid:
    p: 1000.0
    d: 6.0
    i: 600.0
    i_clamp: 4.0

l_elbow_flex_joint:
  joint: l_elbow_flex_joint
  goal_reached_threshold: 0.01
  pid:
    p: 700.0
    d: 4.0
    i: 450.0
    i_clamp: 4.0

l_forearm_roll_joint:
  joint: l_forearm_roll_joint
  goal_reached_threshold: 0.01
  pid:
    p: 300.0
    d: 6.0
    i: 300.0
    i_clamp: 2.0

l_wrist_flex_joint:
  joint: l_wrist_flex_joint
  goal_reached_threshold: 0.01
  pid:
    p: 300.0
    d: 4.0
    i: 300.0
    i_clamp: 2.0

l_wrist_roll_joint:
  joint: l_wrist_roll_joint
  goal_reached_threshold: 0.01
  pid:
    p: 300.0
    d: 4.0
    i: 300.0
    i_clamp: 2.0
